scripts/test/test-utils.sh

```diff
@@ -136,6 +136,7 @@ run_tests() {
 get_pytest_files() {
   wget -N ${MENDER_ACCEPTANCE_URL}/pytest.ini -P $WORKSPACE/mender-image-tests
   wget -N ${MENDER_ACCEPTANCE_URL}/common.py -P $WORKSPACE/mender-image-tests
+  wget -N ${MENDER_ACCEPTANCE_URL}/helpers.py -P $WORKSPACE/mender-image-tests
```

Comment:
from the commit/ PR description I'm not entirely sure, but I assume this is the update to the tests, right?

Reply:
The latest master of the submodule does `from helpers import ...`, which will fail.

This should be failing in master right now the same way, but no builds have been done in the last days.